src/arrayMethodSort.js

```diff
@@ -3,10 +3,40 @@
 /**
  * Implement method Sort
  */
+const compareByDefault = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
```

Comment:
apply early return pattern
```suggestion
  if (stringA > stringB) {
    return 1;
  }
  
  if (stringA === stringB) {
    return 0;
  }
  
  return -1;
```